spec/features/reservations_spec.rb

```diff
@@ -99,7 +99,45 @@
         expect(page).to have_content Time.zone.today.to_s(:long)
         expect(page).to have_content bad_due_date.to_s(:long)
         expect(page).to have_content @eq_model.name
-        expect(page).to have_content 'Create Reservation'
+        expect(page).to have_content 'Confirm Reservation'
+        expect(page).to have_content 'Please be aware of the following errors:'
+        find(:xpath, "//textarea[@id='reservation_notes']").set 'Because'
+        click_button 'Finalize Reservation'
+
+        # check that reservation was created with correct dates
+        query = Reservation.for_eq_model(@eq_model).for_reserver(reserver.id)
+        expect(query.count).to eq(1)
+        expect(query.first.start_date).to eq(Time.zone.today)
+        expect(query.first.due_date).to eq(bad_due_date)
+        expect(query.first.status).to eq('reserved')
+      end
+    end
+
+    shared_examples 'can edit valid reservation' do |reserver|
+      let(:reserver) { reserver }
+      str = reserver ? 'for other user successfully' : 'successfully'
+
+      it str do
+        visit root_path
```

Comment:
So we're still not using this shared example anywhere? I can't find it lower down... Also, this appears to be testing that you can edit the cart through the cart sidebar on the catalog home page, so I'm not sure how it's related to the current issue / PR.